modules/nw-configuring-qinq-sriov-proc.adoc

```diff
@@ -0,0 +1,186 @@
+// Module included in the following assemblies:
+//
+// * 
+// * 
+// * 
+
+:_mod-docs-content-type: PROCEDURE
+
+[id="nw-configuring-qinq-sriov-proc_{context}"]
+= Configuring QinQ support for SR-IOV enabled workloads
+
+
+.Prerequisites
+
+* You installed the OpenShift CLI (`oc`).
+* You have access to the cluster as a user with the `cluster-admin` role.
+* You have installed the SR-IOV Network Operator.
+
+.Procedure
+
+. Create an `SriovNetworkNodePolicy` object, and then save the YAML in the `<name>-sriov-node-network.yaml` file. Replace <name> with the name for this configuration.
++
+[source,yaml]
+----
+apiVersion: sriovnetwork.openshift.io/v1
+kind: SriovNetworkNodePolicy
+metadata:
+  name: sriovnetpolicy-810
+  namespace: openshift-sriov-network-operator
+spec:
+  deviceType: netdevice
+  nicSelector:
+    pfNames:
+      - ens5f0#0-9
+  nodeSelector:
+    node-role.kubernetes.io/worker-cnf: ""
+  numVfs: 10
+  priority: 99
+  resourceName: resource810
+----
+
+. Optional: Label the SR-IOV capable cluster nodes with `SriovNetworkNodePolicy.Spec.NodeSelector` if they are not already labeled. For more information about labeling nodes, see "Understanding how to update labels on nodes".
+
+. Create the `SriovNetworkNodePolicy` object:
++
+[source,terminal]
+----
+$ oc create -f <name>-sriov-node-network.yaml
+----
++
+After applying the configuration update, all the pods in `sriov-network-operator` namespace move to the `Running` status.
+
+. Create a `SriovNetwork` object, setting the outer VLAN referred to as the S-tag, or `Service Tag`, as it belongs to the infrastructure.
+
```

Comment:
Maybe mention that this VLAN needs to be configured on the switch trunk interface.